sdk/communication/Azure.Communication.Identity/samples/Sample1_CommunicationIdentityClient.md

```diff
@@ -63,12 +63,12 @@ Console.WriteLine($"Token: {token}");
 Console.WriteLine($"Expires On: {expiresOn}");
 ```
 
-## Exchange an AAD access token of a Teams User for a Communication Identity access token
+## Exchange an Azure AD access token of a Teams User for a Communication Identity access token
 
-The `CommunicationIdentityClient` can be used to exchange an AAD access token of a Teams user for a new Communication Identity access token with a matching expiration time.
+The `CommunicationIdentityClient` can be used to exchange an Azure AD access token of a Teams user for a new Communication Identity access token with a matching expiration time.
 
 ```C# Snippet:GetTokenForTeamsUser
-Response<AccessToken> tokenResponse = client.GetTokenForTeamsUser(teamsToken);
+Response<AccessToken> tokenResponse = client.GetTokenForTeamsUser(options);
```

Comment:
Can we extend the snippet (`Sample1_CommunicationIdentityClient.cs`) to demonstrate what params need to be passed into the options bag? Or extend the description to explain it in the text? (Similarly to the sample right above it --> Generate a user token)